Frontend/app/mentii.config.ts

```diff
@@ -4,12 +4,23 @@ export class MentiiConfig {
 
   getRootUrl():string {
     var url:string = 'http://api.mentii.me';
```

Comment:
I like that a lot more. I can make the change and test to make sure it works tonight unless someone else gets to it first